feat: add `metrics-computer` cmd

fukusuket opened the pull request:
## What Changed
- Closed #234 

## Evidense
### without -o
<img width="1710" alt="スクリーンショット 2025-03-28 20 08 49" src="https://github.com/user-attachments/assets/7b422c70-fd5b-4dea-9e36-79b581c27e84" />

### with -o
<img width="1710" alt="スクリーンショット 2025-03-28 20 10 49" src="https://github.com/user-attachments/assets/cc4d58d8-3a70-4b21-adc2-81a2a71ac954" />

I would appreciate it if you could check it out when you have time🙏

fukusuket commented:
### automagic
<img width="1710" alt="スクリーンショット 2025-03-28 20 14 54" src="https://github.com/user-attachments/assets/4fc43da3-b5da-4a39-b6a5-0b5ce29ab50c" />

YamatoSecurity reviewed (changes requested):
@fukusuket Thanks so much! The functionality looks perfect, however, can you rename everything `metrics-computers` instead of `computer-metrics`?
I'm sorry I should have explained this more. I also plan to add more "Metrics" commands like `metrics-users` so it would be better to start with `metrics-` so it gets organized better. Even with Hayabusa, I am planning on renaming things `metrics-`, etc.. later on to make it consistent.

fukusuket commented:
@YamatoSecurity 
Sounds good! Thank you for checking! I updated it :)
<img width="1710" alt="スクリーンショット 2025-03-29 7 25 37" src="https://github.com/user-attachments/assets/cd082f96-af57-4ec2-b698-f562cba14875" />

fukusuket commented:
### help
```
fukusuke@fukusukenoMacBook-Air takajo-2.8.0-mac-aarch64 % ./takajo
┏━━━━┳━━━┳┓┏━┳━━━┓ ┏┳━━━┓
┃┏┓┏┓┃┏━┓┃┃┃┏┫┏━┓┃ ┃┃┏━┓┃
┗┛┃┃┗┫┃ ┃┃┗┛┛┃┃ ┃┃ ┃┃┃ ┃┃
  ┃┃ ┃┗━┛┃┏┓┓┃┗━┛┣┓┃┃┃ ┃┃
  ┃┃ ┃┏━┓┃┃┃┗┫┏━┓┃┗┛┃┗━┛┃
  ┗┛ ┗┛ ┗┻┛┗━┻┛ ┗┻━━┻━━━┛
  by Yamato Security

Version: 2.8.0 Ninja Day Releaase
Usage: takajo.exe <COMMAND>

Commands:
  help                           print comprehensive or per-cmd help
  automagic                      automatically executes as many commands as possible and output results to a new folder
  metrics-computer               create a CSV file with computer metrics
  extract-credentials            extract plaintext credentials from the command-line auditing
  extract-scriptblocks           extract and reassemble PowerShell EID 4104 script block logs
  html-report                    create static HTML summary reports for rules and computers with detections
  html-server                    create a dynamic web server to view the HTML summary reports
  list-domains                   create a list of unique domains to be used with vt-domain-lookup
  list-hashes                    create a list of process hashes to be used with vt-hash-lookup
  list-ip-addresses              create a list of unique target and/or source IP addresses to be used with vt-ip-lookup
  list-undetected-evtx           create a list of undetected evtx files
  list-unused-rules              create a list of unused sigma rules
  split-csv-timeline             split up a large CSV file into smaller ones based on the computer name
  split-json-timeline            split up a large JSONL timeline into smaller ones based on the computer name
  stack-cmdlines                 stack executed command lines
  stack-computers                stack computers
  stack-dns                      stack DNS queries and responses
  stack-ip-addresses             stack the target IP addresses (TgtIP field) or source IP addresses (SrcIP field)
  stack-logons                   stack logons by target user, target computer, source IP address and source computer
  stack-processes                stack executed processes
  stack-services                 stack service names and paths from System 7040 and Security 4697 events
  stack-tasks                    stack new scheduled tasks from Security 4698 events and parse out XML task content
  stack-users                    stack target users (TgtUser field) or source users (SrcUser field)
  sysmon-process-tree            output the process tree of a certain process
  timeline-logon                 create a CSV timeline of logon events
  timeline-partition-diagnostic  create a CSV timeline of partition diagnostic events
  timeline-suspicious-processes  create a CSV timeline of suspicious processes
  timeline-tasks                 create a CSV timeline of scheduled tasks
  ttp-summary                    summarize tactics and techniques found in each computer
  ttp-visualize                  extract TTPs and create a JSON file to visualize in MITRE ATT&CK Navigator
  ttp-visualize-sigma            extract TTPs from Sigma and create a JSON file to visualize in MITRE ATT&CK Navigator
  vt-domain-lookup               look up a list of domains on VirusTotal
  vt-hash-lookup                 look up a list of hashes on VirusTotal
  vt-ip-lookup                   look up a list of IP addresses on VirusTotal

Command help: takajo help <COMMAND>

Examples:
  automagic -t ../hayabusa/timeline.jsonl [--level low] [--displayTable] -o case-1
  extract-credentials -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o credentials.csv
  extract-scriptblocks -t ../hayabusa/timeline.jsonl [--level low]  [--skipProgressBar] -o scriptblock-logs
  html-report -t ../hayabusa/timeline.jsonl -o ./output -r ../hayabusa/rules [--sqlite-output html-report.sqlite] [--clobber] [--skipProgressBar]
  html-server -t ../hayabusa/timeline.jsonl [-r ../hayabusa/rules] [-p 8823] [--sqlite-output html-report.sqlite] [--clobber] [--skipProgressBar]
  list-domains -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o domains.txt
  list-hashes -t ../hayabusa/case-1.jsonl [--skipProgressBar] -o case-1
  list-ip-addresses -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o ipAddresses.txt
  list-undetected-evtx -t ../hayabusa/timeline.csv -e ../hayabusa-sample-evtx
  list-unused-rules -t ../hayabusa/timeline.csv -r ../hayabusa/rules
  metrics-computer -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o metrics-computer.csv
  split-csv-timeline -t ../hayabusa/timeline.csv [--makeMultiline] -o case-1-csv
  split-json-timeline -t ../hayabusa/timeline.jsonl -o case-1-json
  stack-cmdlines -t ../hayabusa/timeline.jsonl [--level low] [--skipProgressBar] -o cmdlines.csv
  stack-computers -t ../hayabusa/timeline.jsonl [--level informational] [--sourceComputers] [--skipProgressBar] -o computers.csv
  stack-dns -t ../hayabusa/timeline.jsonl [--level infomational] [--skipProgressBar] -o dns.csv
  stack-ip-addresses -t ../hayabusa/timeline.jsonl [--level infomational] [--targetIpAddresses] [--skipProgressBar] -o ipAddresses.csv
  stack-logons -t ../hayabusa/timeline.jsonl [--skipProgressBar] [--failedLogons] -o logons.csv
  stack-processes -t ../hayabusa/timeline.jsonl [--level low] [--skipProgressBar] -o processes.csv
  stack-services -t ../hayabusa/timeline.jsonl [--level infomational] [--skipProgressBar] -o services.csv
  stack-tasks -t ../hayabusa/timeline.jsonl [--level infomational] [--skipProgressBar] -o tasks.csv
  stack-users -t ../hayabusa/timeline.jsonl [--level infomational] [--sourceUsers] [--filterSystemAccounts] [--filterComputerAccounts] [--skipProgressBar] -o users.csv
  sysmon-process-tree -t ../hayabusa/timeline.jsonl -p <Process GUID> [-o process-tree.txt]
  timeline-logon -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o logon-timeline.csv
  timeline-partition-diagnostic -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o partition-diagnostic-timeline.csv
  timeline-suspicious-processes -t ../hayabusa/timeline.jsonl [--level medium] [--skipProgressBar] [-o suspicious-processes.csv]
  timeline-tasks -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o task-timeline.csv
  ttp-summary -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o ttp-summary.csv
  ttp-visualize -t ../hayabusa/timeline.jsonl [--skipProgressBar] -o mitre-ttp-heatmap.json
  ttp-visualize-sigma -r ../hayabusa/rules -o sigma-rules-heatmap.json
  vt-domain-lookup  -a <API-KEY> --domainList domains.txt -r 1000 -o results.csv --jsonOutput responses.json
  vt-hash-lookup -a <API-KEY> --hashList case-1-MD5-hashes.txt -r 1000 -o results.csv --jsonOutput responses.json
  vt-ip-lookup -a <API-KEY> --ipList ipAddresses.txt -r 1000 -o results.csv --jsonOutput responses.json
```

YamatoSecurity reviewed (approved):
@fukusuket LGTM! Thanks so much!
